tests/cli/test_rasa_train.py

```diff
@@ -194,6 +199,75 @@ def test_train_force(run_in_simple_project_with_model: Callable[..., RunResult])
     assert len(files) == 2
 
 
+def test_train_dry_run(run_in_simple_project_with_model: Callable[..., RunResult]):
+    temp_dir = os.getcwd()
+
+    assert os.path.exists(os.path.join(temp_dir, "models"))
+    files = rasa.shared.utils.io.list_files(os.path.join(temp_dir, "models"))
+    assert len(files) == 1
+
+    output = run_in_simple_project_with_model("train", "--dry-run")
+    printed_output = set(output.outlines)
+
+    assert [s for s in printed_output if "No training required." in s]
+    assert output.ret == 0
+
+
+def test_train_dry_run_failure(
+    run_in_simple_project_with_model: Callable[..., RunResult]
+):
+    temp_dir = os.getcwd()
+
+    domain = (
+        "version: '2.0'\n"
```

Comment:
How about using the constant for 2.0 to avoid that this test causes a warning somewhen in the future?